src/utils/processConcepts.ts

```diff
@@ -50,75 +52,3 @@ export const fetchAndProcessConcepts = async (conceptIds: string[]) => {
 
   return { rootConcepts: rootConceptsResults, concepts: conceptsResults };
 };
-
```

Comment:
Thought: I'm surprised that removing all of this doesn't also come with a bunch of import statements that can also be removed from this file.

Reply:
the only import is for `TConcept`